### PR TITLE
[MM-392]: Fixed the issue of viewcal getting events of the previous day

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**
#### Summary
- Fixed the issue of `/mscalendar viewcal` command getting events of the previous day also
- Updated the help text of the `/mscalendar viewcal` to be more meaningful

#### Ticket Link
  Fixes https://github.com/mattermost/mattermost-plugin-mscalendar/issues/392

##### Existing
Screenshot from 2024-08-14 14-18-10
##### Updated
Screenshot from 2024-08-14 14-16-51

### CODE REVIEW

File: calendar/command/command.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ var cmds = []*model.AutocompleteData{
 			model.NewAutocompleteData("disable", "", "Disable your daily summary."),
 		},
 	},
-	model.NewAutocompleteData("viewcal", "", "View your events for the upcoming week."),
+	model.NewAutocompleteData("viewcal", "", "View your events for the current and upcoming week."),
```
Comment (wiggin77) [inline calendar/command/command.go:57]: @Kshitij-Katiyar  If I ran this command on a Wednesday, would it show events for the Wednesday, Thursday, Friday, and every day the week after?

Comment (raghavaggarwal2308) [reply calendar/command/command.go:57]: @wiggin77 Please refer to this comment for more clarity: https://github.com/mattermost/mattermost-plugin-mscalendar/issues/392#issuecomment-2293011803

**Review by wiggin77 — APPROVED** [review]

**Comment by AayushChaudhary0001** [comment]
@Kshitij-Katiyar Can we change the helper text here?
The current logic says that the events will be fetched for current and upcoming weeks, but if we run this command mid-week, then in general it will be considered starting Sunday/Monday which will not match the logic behind the functionality.
Instead, we can use something which fetches the events for next 14 days/2 weeks starting from today in helper text.

**Comment by Kshitij-Katiyar** [comment]
> @Kshitij-Katiyar Can we change the helper text here? The current logic says that the events will be fetched for current and upcoming weeks, but if we run this command mid-week, then in general it will be considered starting Sunday/Monday which will not match the logic behind the functionality. Instead, we can use something which fetches the events for next 14 days/2 weeks starting from today in helper text.

Hey @AayushChaudhary0001, I have updated the help text to `View your events for the upcoming 14 days, including today.`
let me know if you think this is better.

**Review by AayushChaudhary0001 — APPROVED** [review]
Tested this PR, new helper text looks good, Approved.

**Comment by raghavaggarwal2308** [comment]
@wiggin77 Can you please give this PR another review for changes made by suggestions from by @AayushChaudhary0001

**Comment by wiggin77** [comment]
> if we run this command mid-week, then in general it will be considered starting Sunday/Monday
 
@AayushChaudhary0001 previously when I asked what would happen if the command was run on a Wednesday, the answer was it would include Wed, Thurs, Fri, and then each day of the next week. The statement above seems to contradict that.

> I have updated the help text to `View your events for the upcoming 14 days, including today.`

@Kshitij-Katiyar I cannot approve this until I know exactly what the output will be when running the command on any given day.  IMO, displaying the current day, plus remaining days in current week, plus the following week, was correct.  We can live with just displaying the next 14 days, but I need confirmation that is indeed the behaviour no matter when the command is run.

**Comment by raghavaggarwal2308** [comment]
@wiggin77 It will display the events from current day (When a user runs the command) and the events from 14 days after that. I mentioned the same in my comment [here](https://github.com/mattermost/mattermost-plugin-mscalendar/issues/392#issuecomment-2293011803). 
Please let me know if there is some confusion.

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Pull request merged**